projects/bind9/project.yaml

```diff
@@ -2,11 +2,12 @@ homepage: "https://gitlab.isc.org/isc-projects/bind9"
 language: c
 primary_contact: "security-officer@isc.org"
 auto_ccs:
-  - "ondrej@sury.org"
+  - "ondrej@isc.org"
```

Comment:
let's land this part separately as it seems important. i'll commit it directly to save time

Reply:
https://github.com/google/oss-fuzz/commit/5f1504cb10e1624b2d0098ce39c64562834cd955